source/applications/advanced/hand_eye_calibration/ur_hand_eye_calibration/universal_robots_perform_hand_eye_calibration.py

```diff
@@ -158,19 +159,26 @@ def _get_frame_and_transform_matrix(con: rtde, cam: zivid.Camera):
     return frame, transform
 
 
-def _set_camera_settings(cam: zivid.Camera):
+def _camera_settings() -> zivid.Settings:
     """Set camera settings
 
-    Args:
-        cam: Zivid camera
+    Returns:
+        Zivid Settings
     """
-    with cam.update_settings() as updater:
-        updater.settings.iris = 17
-        updater.settings.exposure_time = datetime.timedelta(microseconds=10000)
-        updater.settings.filters.reflection.enabled = True
-        updater.settings.brightness = 1.0
-        updater.settings.filters.gaussian.enabled = 1
-        updater.settings.gain = 1
+    return zivid.Settings(
+        acquisitions=[
+            zivid.Settings.Acquisition(
+                aperture=8.0, exposure_time=datetime.timedelta(microseconds=10000), brightness=1.0, gain=1,
+            )
+        ],
+        processing=zivid.Settings.Processing(
+            filters=zivid.Settings.Processing.Filters(
+                smoothing=zivid.Settings.Processing.Filters.Smoothing(
+                    gaussian=zivid.Settings.Processing.Filters.Smoothing.Gaussian(enabled=True)
+                )
+            )
+        ),
+    )
```

Comment:
Consider doing this instead:
```
settings = zivid.Settings()
acquisition = zivid.Settings.Acquisition(
                aperture=8.0, exposure_time=datetime.timedelta(microseconds=10000), brightness=1.0, gain=1,
            )
settings.acquisitions.append(acquisition)
settings.processing.filters.smoothing.gaussian.enabled = True
```

It's up to you, just want you (and the customer) to know that it doesn't HAVE to be one-liners when creating Settings :)

Reply:
I will skip this one. This example is focused on a different topic. Showing what you suggest should be done elsewhere I think.

Reply:
Fair enough. Just making you aware of that option.